File: lab6/TripAPI/TripAPI/Models/Liner.cs
namespace CruiseCompany.Models;

public class Liner
{
    public int Id { get; set; }
    public string Name { get; set; } = null!;
    public int Capacity { get; set; }
    public string Class { get; set; } = null!;
    public int YearBuilt { get; set; }

    public ICollection<RouteLiner> RouteLiners { get; set; } = new List<RouteLiner>();
}

File: lab6/app/app/src/main/java/com/example/myapp/model/Liner.java
package com.example.myapp.model;

import java.util.ArrayList;
import java.util.List;

public class Liner {
    public int id;
    public String name;
    public int capacity;
    public String clazz;
    public int yearBuilt;
    public List<CruiseRoute> routes = new ArrayList<>();
}